### PR TITLE
Change package and Maven groupId to app.cash

### PR DESCRIPTION
**Pull request by JakeWharton**
CashApp Inflation Inject by CashApp™

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+POM_SCM_DEV_CONNECTION=scm:git:ssh://git@github.com/cashapp/InflationInjection.git
 
 POM_LICENCE_NAME=The Apache Software License, Version 2.0
 POM_LICENCE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_LICENCE_DIST=repo
 
-POM_DEVELOPER_ID=square
-POM_DEVELOPER_NAME=Square, Inc.
+POM_DEVELOPER_ID=cashapp
+POM_DEVELOPER_NAME=CashApp
```
Comment (mattprecious) [inline gradle.properties:16]: Is this what we use elsewhere? Without the space?

**Review by oldergod — APPROVED** [review]

**Review by mattprecious — APPROVED** [review]
CashApp Approval by CashApp

File: gradle.properties (hunk 1 above)
Comment (JakeWharton) [reply gradle.properties:16]: No idea.

SQLDelight DGAF
https://github.com/cashapp/sqldelight/blob/90204d0d047075b34395c4b49a708f8379b40698/gradle.properties#L13-L14

Turbine (also by me):
https://github.com/cashapp/turbine/blob/a710b59dc8010e952611a5c2ff0882e9311b63f9/gradle.properties#L20-L21

Are you saying it needs more ™™™™™?

Comment (JakeWharton) [reply gradle.properties:16]: We need to decide now or it's going to get cargo culted to all hell...

Comment (JakeWharton) [reply gradle.properties:16]: It can be changed at any time so... merging.

**Pull request merged**